src/ralph/ssl_certificates/admin.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+
+from django.utils.translation import ugettext_lazy as _
+
+from ralph.admin import RalphAdmin, register
+from ralph.admin.filters import DateListFilter
+from ralph.attachments.admin import AttachmentsMixin
+from ralph.ssl_certificates.models import SSLCertificate
+
+
+@register(SSLCertificate)
+class SSLCertificateAdmin(AttachmentsMixin, RalphAdmin):
+    list_select_related = [
+        'technical_owner', 'business_owner',
+        'issued_by'
+    ]
+
+    list_filter = [
+        'name', 'certificate_type', 'business_owner',
+        'technical_owner', ('date_from', DateListFilter),
+        ('date_to', DateListFilter), 'issued_by',
```

Comment:
please keep consistent presence or absence of `,` after last item

Reply:
Sorry for that. I'll fix it